app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/coffee-table-groups.mdx

```diff
@@ -0,0 +1,169 @@
+---
+meta:
+  title: Coffee Table Groups
+  description: The list of Coffee Table Groups — the small, special interest groups created and run by our members.
+hero:
+  Hero: UndrawConversation
+tags: [memberresources]
+order: 4
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+import { Link } from '@remix-run/react';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+Coffee Table Groups are small, special interest groups created and run by our members. They can involve Zoom meetings, async Slack hangouts, or anything else the members want to do.
+
+In this article, you will find the list of our current Coffee Table Groups based on the day and time of the events. You can contact the group's Leader for any questions or ask for more information, and you are most welcome to join!
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
+## Tech Interview Study Group
+
+<LeadText>
+
+_Mondays at 4:00 PM ET_
+
+**Leaders: Julia Seidman, Marie Antons**
+
+</LeadText>
+
+A weekly discussion of job search & interview-related topics — mini mock interviews, technical topic discussions, job search tips, occasional guest speakers, etc. We are always looking for ideas to help meet your needs! All stages of career & job search are welcome.
+
+Interested? Join the `#tech-interview-study-group` Slack channel!
+
+</TextContainer>
+
+<TextContainer>
+
+## Accountabilibuddies
+
+<LeadText>
+
+_Tuesdays at 7:00 PM ET, Thursdays at 9:00 AM ET, and every other Sunday at 1:00 PM ET_
+
+**Leaders: <br />
+Tuesdays: Lillian Chan, Sadie Jay | Thursdays: Meg Gutshall, Joe Karow | Sundays: Raynald Mirville**
```

Comment:
👏 👏 👏 Love it. If you wanted to update the leaders sections in [VC Contributors](https://github.com/Virtual-Coffee/VC-Contributors/blob/main/coffee-table-groups/coffee-table-groups.md) to match, that would be great.

Reply:
Thanks, Abbey! Sure, I'll do that 😊